### PR TITLE
chore(release): 2.4.1

### PR DESCRIPTION
**Pull request by marciadasilva**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+### Bug Fixes
+
+- adjust order cancel schema ([#65](https://github.com/rewardops/rewardops-sdk-node/issues/65)) ([c8eae65](https://github.com/rewardops/rewardops-sdk-node/commit/c8eae65739cf37467f1c59dddd741ebe9f76116a))
+
+### [2.4.0](https://github.com/rewardops/rewardops-sdk-node/compare/v2.3.0...v2.4.0) (2021-01-19)
+
+### Features
+
+- add new `cancelOrders` function ([#64](https://github.com/rewardops/rewardops-sdk-node/issues/64)) ([f6fd846](https://github.com/rewardops/rewardops-sdk-node/commit/f6fd8465f8ead2b8969cd6cf58630a1c74e50e78))
+- add request object to the api callback signature ([#61](https://github.com/rewardops/rewardops-sdk-node/issues/61)) ([0c0ed77](https://github.com/rewardops/rewardops-sdk-node/commit/0c0ed77ffb6539a54cad1ac022b82e548ef45c94))
```
Comment (Alex-Joseph) [inline CHANGELOG.md:16]: Comment: I think this is a duplicate of line 27. If so, can we remove it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+
+### Bug Fixes
+
+- catch all 5XX and 4XX statusCodes ([#56](https://github.com/rewardops/rewardops-sdk-node/issues/56)) ([637a5e8](https://github.com/rewardops/rewardops-sdk-node/commit/637a5e841368e4b721e77567414cbd54eff9a732))
+- create instance of axios for PII calls ([#57](https://github.com/rewardops/rewardops-sdk-node/issues/57)) ([1ac41cf](https://github.com/rewardops/rewardops-sdk-node/commit/1ac41cf48bdbca1673df2051360a5d3e107d3476))
+
```
Comment (Alex-Joseph) [inline CHANGELOG.md:22]: Comment: I think this whole block is a duplicate of line 29-32

**Review by shamrt — APPROVED** [review]

**Review by Alex-Joseph — CHANGES_REQUESTED** [review]
Comment: I think there is some duplication in the changelog 🤔

**Comment by shamrt** [comment]
Closed due to conflict with #67

**Pull request closed without merging**